packages/mobile/src/store/offline-downloads/sagas/redownloadStaleTracksSaga.ts

```diff
@@ -4,7 +4,7 @@ import { select } from 'typed-redux-saga'
 import { getOfflineTrackMetadata } from '../selectors'
 import type { OfflineItem } from '../slice'
 
-const STALE_DURATION_TRACKS = moment.duration(1, 'second')
```

Comment:
whoops!

Reply:
Also wondering if this should be longer? What's the cost of sync for a large library?

Reply:
so i have batched to only 50? so itll be 50 per session ad infinum probably. and keep in mind we only redownload if created_at changes, so these fly by pretty quick. but def up for whatever you think.

Reply:
Yeah that sounds good. Wondering if it's worth sorting to start with the most stale first? If the queue is just that fast though it's probably irrelevant.

If weekly session count is less than CollectionSize / 50 then we may never catch up. Not sure fairness is helpful there though. There may be legal requirements for how often we have to guarantee check each track, so maybe a follow-up could be doing 50/session plus anything over 30d gets sync'd immediately. Idk

Reply:
ah yeah that is smart about most stale! love that idea, will add that now, and then we can follow up with any more things we thing makes sense